eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/persistence/DbsnpClusteredVariantEntity.java

```diff
@@ -0,0 +1,141 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.dbsnp.persistence;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.ac.ebi.ampt2d.commons.accession.persistence.mongodb.document.AccessionedDocument;
+
+import uk.ac.ebi.eva.accession.core.ClusteredVariant;
+import uk.ac.ebi.eva.accession.core.IClusteredVariant;
+import uk.ac.ebi.eva.accession.core.VariantType;
+
+import java.util.Arrays;
+
+@Document
+public class DbsnpClusteredVariantEntity extends AccessionedDocument<Long> implements IClusteredVariant {
+
+    @Field("asm")
+    private String assemblyAccession;
+
+    @Field("tax")
+    private int taxonomyAccession;
+
+    private String contig;
+
+    private long start;
+
+    private VariantType type;
+
+    @Field("validated")
+    private Boolean validated;
```

Comment:
Shouldn't we set the 'validated' default (let say false) to null to save space as we did with the SSs?